### ISSUE TITLE
Incorrect default value inferring for stateless components

### ISSUE THREAD

**Issue by arunoda** [reactjs/react-docgen#131]
This issue came with the version `2.12.0`.
And we got this while working with our [babel plugin for doc gen](https://github.com/kadirahq/babel-plugin-react-docgen).

Assume we've a stateless component like this:

```js
const Button = ({ children, onClick, style = {} }) => (
  <button
    style={{ }}
    onClick={onClick}
  >
    {children}
  </button>
);

Button.propTypes = {
  children: React.PropTypes.string.isRequired,
  onClick: React.PropTypes.func,
  style: React.PropTypes.object,
};

export default Button;
```

Then generated info like this:

```js
{
  description: '',
  props: {
    children: {
      type: {
        name: 'string'
      },
      required: true,
      description: '',
      defaultValue: {
        value: 'children',
        computed: true
      }
    },
    onClick: {
      type: {
        name: 'func'
      },
      required: false,
      description: '',
      defaultValue: {
        value: 'onClick',
        computed: true
      }
    },
    style: {
      type: {
        name: 'object'
      },
      required: false,
      description: '',
      defaultValue: {
        value: '{}',
        computed: false
      }
    }
  }
}
```

Just look at the default values for `onClick` or `children`.
This doesn't seems to be correct to me.

**Comment by fkling** [comment]
Thanks, I'll look into it!

**Comment by fkling** [comment]
This should be fixed with v2.12.1!

**Comment by arunoda** [comment]
This is pretty fast guys. Awesome.
